### PR TITLE
CMCL-0000 Fix typos in documentation

### PR DESCRIPTION
**Pull request by AntoineCharton**
### Purpose of this PR

Fix some typos in docs

### CODE REVIEW

File: com.unity.cinemachine/Runtime/Behaviours/CinemachineConfiner2D.cs [hunk 1]
```diff
@@ -296,7 +296,7 @@ static float CalculateHalfFrustumHeight(in CameraState state, in float cameraPos
                 frustumHeight = state.Lens.OrthographicSize;
             else
             {
-                // distance between the collider's plane and the camera
+                // distance between the collider plane and the camera
```
Comment (gaborkb) [inline com.unity.cinemachine/Runtime/Behaviours/CinemachineConfiner2D.cs:299]: I think this was ok. Or maybe `plane of the collider`.

File: com.unity.cinemachine/Runtime/Behaviours/CinemachineDeoccluder.cs [hunk 2]
```diff
@@ -260,7 +260,7 @@ public void AddPointToDebugPath(Vector3 p)
 #endif
             }
 
-            // Thanks to Sebastien LeTouze from Exiin Studio for the smoothing idea
+            // Thanks to Sebastien LeTouze from Exiting Studio for the smoothing idea
```
Comment (gaborkb) [inline com.unity.cinemachine/Runtime/Behaviours/CinemachineDeoccluder.cs:263]: Not sure if this is correct

File: com.unity.cinemachine/Runtime/Components/CinemachineRotationComposer.cs [hunk 3]
```diff
@@ -187,7 +187,7 @@ public override void MutateCameraState(ref CameraState curState, float deltaTime
             {
                 if (deltaTime >= 0 && VirtualCamera.PreviousStateIsValid)
                     curState.RawOrientation = m_CameraOrientationPrevFrame;
-                return;  // navel-gazing, get outa here
```
Comment (gaborkb) [inline com.unity.cinemachine/Runtime/Components/CinemachineRotationComposer.cs:190]: I think this is useful, `get outa here` may be not.

**Review by gaborkb — CHANGES_REQUESTED** [review]
👍 

Small change requests only.

File: com.unity.cinemachine/Runtime/Behaviours/CinemachineConfiner2D.cs (hunk 1 above)
Comment (glabute) [reply com.unity.cinemachine/Runtime/Behaviours/CinemachineConfiner2D.cs:299]: nobody cares, it's private in-line doc :)

File: com.unity.cinemachine/Runtime/Behaviours/CinemachineDeoccluder.cs (hunk 2 above)
Comment (glabute) [reply com.unity.cinemachine/Runtime/Behaviours/CinemachineDeoccluder.cs]: Please revert this change

File: com.unity.cinemachine/Runtime/Components/CinemachineRotationComposer.cs (hunk 3 above)
Comment (glabute) [reply com.unity.cinemachine/Runtime/Components/CinemachineRotationComposer.cs:190]: please revert this change

File: com.unity.cinemachine/Runtime/Behaviours/CinemachineDeoccluder.cs (hunk 2 above)
Comment (AntoineCharton) [reply com.unity.cinemachine/Runtime/Behaviours/CinemachineDeoccluder.cs]: Fixed

**Review by glabute — CHANGES_REQUESTED** [review]

File: com.unity.cinemachine/Runtime/Behaviours/CinemachineConfiner2D.cs (hunk 1 above)
Comment (AntoineCharton) [reply com.unity.cinemachine/Runtime/Behaviours/CinemachineConfiner2D.cs:299]: Anyway not public shouldn't have changed it mb

File: com.unity.cinemachine/Runtime/Components/CinemachineRotationComposer.cs (hunk 3 above)
Comment (AntoineCharton) [reply com.unity.cinemachine/Runtime/Components/CinemachineRotationComposer.cs:190]: Not sure I understand what it means what is navel-gazing? Could we replace it with at least something more technically accurate?

Comment (glabute) [reply com.unity.cinemachine/Runtime/Components/CinemachineRotationComposer.cs:190]: It's appropriate here.  Google it..

File: com.unity.cinemachine/Runtime/Core/CameraState.cs [hunk 4]
```diff
@@ -104,7 +104,7 @@ public enum BlendHintValue
         }
 
         /// <summary>
-        /// These hints can be or'ed toether to influence how blending is done, and how state
```
Comment (glabute) [inline com.unity.cinemachine/Runtime/Core/CameraState.cs:107]: This is useful info, why did you delete it?

File: com.unity.cinemachine/Runtime/Components/CinemachinePositionComposer.cs [hunk 5]
```diff
@@ -130,7 +130,7 @@ float CinemachineFreeLookModifier.IModifiableDistance.Distance
         /// Always returns the Body stage</summary>
         public override CinemachineCore.Stage Stage => CinemachineCore.Stage.Body;
 
-        /// <summary>FramingTransposer's algorithm tahes camera orientation as input, 
+        /// <summary>FramingTransposer algorithm taKes camera orientation as input,
```
Comment (glabute) [inline com.unity.cinemachine/Runtime/Components/CinemachinePositionComposer.cs:133]: typo

File: com.unity.cinemachine/Runtime/Components/CinemachineRotationComposer.cs (hunk 3 above)
Comment (AntoineCharton) [reply com.unity.cinemachine/Runtime/Components/CinemachineRotationComposer.cs:190]: Reverted it anyway it's a comment so it just concerns the few who will check the code

File: com.unity.cinemachine/Runtime/Core/CinemachineCore.cs [hunk 6]
```diff
@@ -382,7 +382,7 @@ internal enum UpdateFilter
         {
             Fixed = UpdateTracker.UpdateClock.Fixed,
             Late = UpdateTracker.UpdateClock.Late,
-            Smart = 8, // meant to be or'ed with the others
+            Smart = 8, // meant to be sorted with the others
```
Comment (glabute) [inline com.unity.cinemachine/Runtime/Core/CinemachineCore.cs:385]: please revert this change

File: com.unity.cinemachine/Runtime/Core/CinemachineExtension.cs [hunk 7]
```diff
@@ -68,7 +68,7 @@ protected virtual void ConnectToVcam(bool connect)
             m_ExtraState = null;
         }
 
-        /// <summary>Override this to do such things as offset the RefereceLookAt.
+        /// <summary>Override this to offset the ReferenceLookAt.
```
Comment (glabute) [inline com.unity.cinemachine/Runtime/Core/CinemachineExtension.cs:71]: Please revert this change

**Review by glabute — CHANGES_REQUESTED** [review]

File: com.unity.cinemachine/Runtime/Core/CameraState.cs (hunk 4 above)
Comment (AntoineCharton) [reply com.unity.cinemachine/Runtime/Core/CameraState.cs:107]: I could put it back yes. or'ed is sorted?

File: com.unity.cinemachine/Runtime/Core/CinemachineCore.cs (hunk 6 above)
Comment (AntoineCharton) [reply com.unity.cinemachine/Runtime/Core/CinemachineCore.cs:385]: What is or'ed?

File: com.unity.cinemachine/Runtime/Core/CameraState.cs (hunk 4 above)
Comment (glabute) [reply com.unity.cinemachine/Runtime/Core/CameraState.cs:107]: No. (A | B) is A and B or'ed together

File: com.unity.cinemachine/Runtime/Core/CinemachineCore.cs (hunk 6 above)
Comment (AntoineCharton) [reply com.unity.cinemachine/Runtime/Core/CinemachineCore.cs]: Reverted

File: com.unity.cinemachine/Runtime/Components/CinemachinePositionComposer.cs (hunk 5 above)
Comment (AntoineCharton) [reply com.unity.cinemachine/Runtime/Components/CinemachinePositionComposer.cs]: Fixed

File: com.unity.cinemachine/Runtime/Core/CinemachineExtension.cs (hunk 7 above)
Comment (AntoineCharton) [reply com.unity.cinemachine/Runtime/Core/CinemachineExtension.cs]: Reverted the removal.

File: com.unity.cinemachine/Runtime/Core/InputAxis.cs [hunk 8]
```diff
@@ -211,10 +212,10 @@ struct RecenteringState
         RecenteringState m_RecenteringState;
 
         /// <summary>
-        /// Call this before calling DoRecentering.  Will track any vlue changes so that the recentering clock
+        /// Call this before calling DoRe-centering.  Will track any value changes so that the re-centering clock
```
Comment (glabute) [inline com.unity.cinemachine/Runtime/Core/InputAxis.cs:215]: DoRecentering is a method name, it should be left alone

**Review by glabute — CHANGES_REQUESTED** [review]

File: com.unity.cinemachine/Runtime/Core/InputAxis.cs (hunk 8 above)
Comment (AntoineCharton) [reply com.unity.cinemachine/Runtime/Core/InputAxis.cs]: Fixed

**Review by glabute — APPROVED** [review]

**Review by gaborkb — APPROVED** [review]
